Source/Particles/Deposition/ChargeDeposition.H

```diff
@@ -18,19 +18,40 @@
 
 #include <AMReX.H>
 
+#if defined(AMREX_USE_GPU)
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+void addLocalToGlobal (const amrex::Box& bx,
+                       const amrex::Array4<amrex::Real>& global,
+                       const amrex::Array4<amrex::Real>& local) noexcept
+{
+    const auto lo  = amrex::lbound(bx);
+    const auto len = amrex::length(bx);
+    for (int icell = threadIdx.x; icell < bx.numPts(); icell += blockDim.x)
```

Comment:
Intel DPC++ doesn't know what to do with any of these CUDA-ish constructs. Should `if defined(AMREX_USE_GPU)` actually be `CUDA || HIP` throughout? Or is there an easy adaptation to be made?

Reply:
Yep, this is the way.